src/Middleware/UnauthorizedHandler/CakeRedirectHandler.php

```diff
@@ -25,6 +25,7 @@
  * This handler will redirect the response if one of configured exception classes is encountered.
  *
  * CakePHP Router compatible array URL syntax is supported.
+ * @deprecated use RedirectHandler instead.
```

Comment:
:clap: I agree with the service hook method developers can use `Router::url()` to generate the URL they need.